### DISCUSSION TITLE
doris 2.0.1 使用catalog 读取hive表一开始能成功读取，在insert overwrite之后报错

### DISCUSSION THREAD

**Discussion by sjtianwen** [apache/doris#24704] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/50941338/86b64d4c-c417-4509-893d-9d4f9c271bbc)
使用
REFRESH CATALOG hive PROPERTIES("invalid_cache" = "true");
仍然不能够正常读取，但是如果重启doris集群就可以正常读取，希望能够获得不重启集群能正常读取hive insertoverwrite之后的表的解决办法

**Comment by yagagagaga** [comment]
Could you provide complete fe and be logs?

**Reply by sjtianwen** [reply]
[CANCELLED][INTERNAL_ERROR]Read parquet file /user/hive/warehouse/ads.db/ads_quna_new_machine_all2/dt=2023-09-20/000000_0 failed, reason = [IO_ERROR]Failed to deserialize parquet page header. offset: 424, header size: 612, end offset: 1036, real header size: 612

**Reply by yagagagaga** [reply]
Can you provide more context for this log? If it's not convenient, you can add me as your WeChat contact. I will provide more support.
![](https://user-images.githubusercontent.com/21151126/222130177-a85c1e2b-bdaa-4c96-98bc-2c4c8da02615.JPG)

**Reply by yagagagaga** [reply]
This has been fixed in 2.0.2
